### ISSUE TITLE
Creating room in particular space

### ISSUE THREAD

**Issue by frost-cqre** [SAPUCC/inviterbot#11]
Is it possible to create a room in particular space?

**Comment by v411e** [comment]
No, I think this is not possible yet. You would have to add the room manually to the space. But it's an interesting idea.
